app/objects/secondclass/c_fact.py

```diff
@@ -18,16 +20,28 @@
 }
 
 
+class FactSchema(ma.Schema):
+
+    unique = ma.fields.String()
+    trait = ma.fields.String()
+    value = ma.fields.String()
```

Comment:
Should all values be strings?
If so, how should we enforce in the sources or in the loading?

Reply:
Pretty sure these are all strings. 

In the source schema, this schema is referenced as a nested schema https://github.com/mitre/caldera/pull/1465/files#diff-bd4c47907082a94d1fed244697c6233dR29 .  This means that the source schema will validate each nested fact entry during load.  If a non-string value was here, the schema would raise a ValidationError.  

This PR also changes data_svc to load sources with the schema, so loading sources from yaml will be subject to the validation https://github.com/mitre/caldera/pull/1465/files#diff-ea3235067950e38535daba9c061b1817R244

Reply:
say I put a fact value of 8080 in a source it would fail this validation. yaml would interpret this as an int and would require putting in as '8080' or "8080" as the fact value.

another option is casting all fact values to str or allowing multiple data types on the values in the schema (if possible)
thoughts?

Reply:
this will coerce fact values into strings when they're deserialized https://github.com/mitre/caldera/pull/1465/commits/0a0c4af7f74fa5c9ff28ed733e3ed6e0868ee67b